src/gui/optionsdialog.cpp

```diff
@@ -1458,7 +1458,6 @@ void OptionsDialog::enableProxy(int index)
         else {
             m_ui->checkProxyAuth->setEnabled(false);
             m_ui->checkProxyAuth->setChecked(false);
```

Comment:
I think this line should be removed too. As we don't uncheck it in the case of using no proxy.

Reply:
Yes this was what i intended to remove but i accidentally removed the line below.